organising/organising-a-meeting.md

```diff
@@ -0,0 +1,164 @@
+
+* one 10 minute talk, one 25 minute talk, one 40 minute talk (~75 minutes)
+* three 25 minute talks (~75 minutes)
+* two 25 minute slots, two 10 minute slots (~70 minutes)
+
+Combinations that work, but are painful:
+
+* two 40 minute talks (~80 minutes) - 40 minute talks can drag on a bit, we try to avoid this, but sometimes this is what we
+
+Special combinations:
+
+* eight 10 minute talks (~80 minutes) - usually this is what we do for our February lightning talks only event.  It's a fun evening, but tough to organise so we don't do it that often.  If you have a glut of speakers though, by all means!
+* one 80 minute session - for a practical event, or panel discussion, or pub quiz, or something.  We wouldn't want a single speaker to talk for 80 minutes, but if someone has an idea for a interactive session that would take up the whole time then why not try it!
+
+But!  Any combination that fills roughly 80 minutes will do.  It's also fine to not fill 80 minutes.  Getting one speaker in just so we can make an LRUG happen is good.
+
+Speakers often don't fill their allotted time, so we frequently end before our self-imposed 8pm curfew.
+
+### What if it doesn't go smoothly
+
+Hopefully all that is needed is for you to pick the top 2 or 3 volunteers who are already earmarked for the meeting you are organising and email them to confirm the date and their timeslot.  They get back to you and the meting is full of speakers.
+
+It doesn't always go that smoothly.
+
+* Speakers pull out
+* Speakers just don't respond
+* We don't have anyone earmarked for that meeting
+
+In this case you'll need to actually find some speakers as well as confirm them.  Check out [our tips on populating the speaker backlog](/organising/maintaining-the-community#populate-the-speaker-backlog) if you need to find speakers.  Adding an urgent "_for this month's meeting_" plea can help.
+
+## Announce the meeting
+
+Once you have at least one speaker confirmed you are ready to announce the meeting.  This means:
+
+1. [Set up the eventbrite page so people get reserve their place](/organising/organising-a-meeting#set-up-the-eventbrite-page)
+2. [Schedule the zoom meeting](/organising/organising-a-meeting#schedule-the-zoom-meeting)
+3. [Write up the meeting on lrug.org](/organising/organising-a-meeting#write-up-on-lrugorg)
+4. [Tell the mailing list about it](/organising/organising-a-meeting#tell-the-mailing-list-about-the-meeting)
+5. [Tweet about it](/organising/organising-a-meeting#tweet-about-the-meeting)
+
+Ideally we do all this the week after the previous meeting, but sometimes we do it really close to the event if we're struggling to get speakers.  It can be useful to write up placeholder versions and publish them, so that people will at least get the event into their calendars.
+
+If you have more than one speaker it can be useful to trickle the speaker announcements out over a couple of weeks in order to keep reminding people.
```

Comment:
```suggestion
If we have more than one speaker it can be useful to trickle the speaker announcements out over a couple of weeks in order to keep reminding people.
```